packages/expo-updates/ios/EXUpdates/UpdatesUtils.swift

```diff
@@ -91,7 +91,11 @@ public final class UpdatesUtils: NSObject {
 
   internal static func shouldCheckForUpdate(withConfig config: UpdatesConfig) -> Bool {
     func isConnectedToWifi() -> Bool {
-      return try! Reachability().connection == .wifi
+      do {
```

Comment:
were you noticing exceptions here?

Reply:
This is to fix a SwiftLint violation. My solution was to return false if Reachability threw any exception